### ISSUE TITLE
Adding a reaction (:+1:) vs a comment (:+1:) doesn't complete signoff

### ISSUE THREAD

**Issue by n8io** [paultyng/further-review#30]
![image](https://cloud.githubusercontent.com/assets/2074517/18091065/5ed794e4-6e95-11e6-9b76-806e5bd3b27d.png)

This may not be a bug. It might just be some training on how to signoff a PR too.

**Comment by paultyng** [comment]
Yeah, I was considering this in #4, I think we should just go ahead and add it.

**Comment by paultyng** [comment]
They are different things in the Github API, so it has to be queried separately, thats the only reason its not working yet.

**Comment by CKarper** [comment]
I would not like this, as I use reactions to approve/disapprove specific changes in convo.

**Comment by kjvalencik** [comment]
@Clapper I would only use a reaction on the PR, not comments.
